webapp-django/crashstats/crashstats/tests/test_views.py

```diff
@@ -1440,13 +1440,15 @@ def mocked_raw_crash_get(**params):
             assert "datatype" in params
             if params["datatype"] == "meta":
                 crash = copy.deepcopy(_SAMPLE_META)
-                crash["TelemetryEnvironment"] = {
-                    "key": ["values"],
-                    "plainstring": "I am a string",
-                    "plainint": 12345,
-                    "empty": [],
-                    "foo": {"keyA": "AAA", "keyB": "BBB"},
-                }
+                crash["TelemetryEnvironment"] = json.dumps(
+                    {
+                        "key": ["values"],
+                        "plainstring": "I am a string",
+                        "plainint": 12345,
+                        "empty": [],
+                        "foo": {"keyA": "AAA", "keyB": "BBB"},
+                    }
+                )
```

Comment:
This test was bad. The value for `TelemetryEnvironment` (and all annotations) is always a string.